misc/scripts/deploy-tools.sh

```diff
@@ -210,6 +211,8 @@ function deployESXInstall {
 }
 
 function deployESXPost {
+    # restart hostd so that esxcli extension file is read
+    $SSH $TARGET $HOSTD restart
```

Comment:
1. how about through `esxcli`? 
2. https://github.com/vmware/docker-volume-vsphere/pull/1735#discussion_r132777695 : it is a requirement of hostd restart then README and installation guideline should be updated as part of this review.